src/Home/ActivationStatus.tsx

```diff
@@ -25,38 +25,59 @@ export const ActivationStatus: FunctionComponent<ActivationStatusProps> = ({
 }) => {
   const { t } = useTranslation()
 
-  const bodyText = isActive ? t("common.enabled") : t("common.disabled")
-  const icon = isActive ? Icons.CheckInCircle : Icons.XInCircle
-  const iconFill = isActive ? Colors.success100 : Colors.danger75
+  type Content = {
```

Comment:
maybe the type definition could live outside the component body since this will be transpiled away when we turn it into javascript?

Reply:
Good call!